genesis-programs/src/lib.rs

```diff
@@ -143,11 +181,6 @@ pub fn get_entered_epoch_callback(operating_mode: OperatingMode) -> EnteredEpoch
                 }
             }
         }
-        if OperatingMode::Stable == operating_mode {
-            bank.set_cross_program_support(bank.epoch() >= 63);
-        } else {
-            bank.set_cross_program_support(true);
-        }
 
         bank.set_max_invoke_depth(DEFAULT_MAX_INVOKE_DEPTH);
```

Comment:
Ryo, if easy can you also move these to the bank like you did for `cross-program-support`.  If not easy let me know and I'll open a new pr